pom.xml

```diff
@@ -252,6 +252,11 @@
                 <version>${powermock.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
```

Comment:
Let's not include this with test dependencies. Add this with logging dependencies.